src/memjs/memjs.ts

```diff
@@ -333,10 +342,10 @@ class Client<Value = MaybeBuffer, Extras = MaybeBuffer> {
       // after the first response. Logic in server.js.
       handle.quiet = true;
 
-      const request = this._buildGetMultiRequest(keys);
+      const seq = this.incrSeq();
```

Comment:
Would it be helpful to audit usage of `this.seq` by converting all calls to `this.incrSeq`?

Reply:
probably, which is basically what your suggestion above would force us to do, right? since this.seq would no longer be accessible anywhere?

Reply:
yep. wrote the comment before the suggestion